coda/coda_mdstore/views.py

```diff
@@ -639,10 +641,17 @@ def bagProxy(request, identifier, filePath):
     handle = getFileHandle(identifier, filePath)
     if handle:
         resp = HttpResponse(
-            FileWrapper(handle),
             content_type=handle.info().getheader('Content-Type')
         )
         resp['Content-Length'] = handle.info().getheader('Content-Length')
+        if hasattr(settings, 'REPROXY') and settings.REPROXY:
+            # Have a proxy server point the client to where to download
+            # the file directly in order to bypass serving through Django.
+            resp['X-REPROXY-URL'] = handle.geturl()
+            resp['ETag'] = '"%s"' % uuid.uuid4().get_hex()
```

Comment:
I'm a little confused about the use of quotes here.

Reply:
`ETag` header values are specified to be in [double quotes](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/ETag#Directives). Is that what you are wondering about?

Reply:
yep, that was it. thanks for clearing that up!